packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/ConfigurationFiles/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import {
+  Element,
+  Collapsible,
+  Text,
+  Button,
+  Stack,
+  Grid as BaseGrid,
+} from '@codesandbox/components/lib/';
```

Comment:
I think you don't have to mention `/lib/` anymore. because the `package.main` points to `lib/index`

Reply:
Trueeeee, done!